### ISSUE TITLE
Some downloads don't show up on the bar

### ISSUE THREAD

**Issue by braille8011** [jwensley2/DownloadStatusBar#32]
Occasionally a download doesn't show up on the bar when initiated, even though I can see the download in progress in the default Downloads list.

I have not found a consistent cause for the issue. For instance, I've seen it happen whether I have "Force downloads to stay in a single row" enabled or disabled.

**Comment by jwensley2** [comment]
I can confirm that this is an issue but I haven't found a way to reproduce it so I'm not sure of the cause, it could be in the addon or in Firefox itself.

Hopefully I can figure out a way to reproduce it so I can fix it or confirm that it's a Firefox bug.

**Comment by braille8011** [comment]
Thanks for letting me know it may not be just me. Unfortunately, the issue
just got worse for me. I haven't seen the status bar pop up for the last 20
downloads or so in a row. Before it was usually only 1 of 10 that failed to
go to the bar. If there is anything I can do to help troubleshoot the
issue, please let me know. I hated that the recent Firefox update killed
all of the old addons and this is the first download status bar I've seen
that's built to work with the new Firefox.

**Comment by braille8011** [comment]
Update: Restarted Firefox and now it's back to only about 1 of 10 downloads failing to show up.

**Comment by Sopor-** [comment]
Yes, i have noticed that too and a restart of Firefox will make it work again.

**Comment by topoldo** [comment]
If you try to download .xpi files (for example extensions for Thunderbird beside to those for FF) then you do not have them in the Download Statusbar. Maybe you excluded this extension by default?
Topoldo
